Runtime/Components/ConeRayAnglesEstimator/PeakConeRaySegmentComputation.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using ubco.ovilab.HPUI.Interaction;
+using UnityEngine.Assertions;
+
+namespace ubco.ovilab.HPUI
+{
+    /// <summary>
+    /// Computes the cone rays from the frames that have the minimum ray distance in a given <see cref="ConeRayComputationDataRecord"/>.
+    /// </summary>
+    [Serializable]
+    public class PeakConeRaySegmentComputation : IConeRaySegmentComputation
```

Comment:
Peak is a confusing name

Reply:
Open to suggestions :D

Reply:
MinConeRaySegmentComputation

Reply:
I am ok with this, @ThatAmuzak can fix this?